app/components/review_component/review_component.scss

```diff
@@ -1,5 +1,17 @@
 @import 'base_component';
 
+ol {
```

Comment:
this looks odd, shouldnt use tag names as selectors if possible, use a class put on them